app/components/SummaryPrintTable/index.tsx

```diff
@@ -12,6 +12,15 @@ type SummaryPrintTableProps = {
   renderValue?: (value: any) => unknown;
 };
 
+const variantTypes = ['cnv', 'smallMutation', 'structuralVariant', 'expression'];
+
+const variantToStrings: Dictionary<string> = {
```

Comment:
weird, was the import from lodash needed? I thought the type inferrence by typescript was enough

Reply:
oh yea I tried the native typescript dict but it did not let me index with strings so I had to use the lodash dict